packages/components/templates/f-template-subnav/README.md

```diff
@@ -99,15 +99,13 @@ Example #2 (*navigation-links*)
 [
   {
     id: 'accountNavLinkInfo'  // Test Data Id
-    url: '/account/info',     // Url
+    to: '/account/info',      // `href` for `<a>`, or `to` for `<router-link>`
```

Comment:
Is it worth added a one line note near the heading of this example around this? Just so it clear  to the user the`href `& `to` are interchangeable.